tests/test_stac.py

```diff
@@ -91,3 +91,35 @@ def test_create_item_xml_grid_code(self):
         self.assertEqual(
             item.datetime, str_to_datetime("20200902") + timedelta(hours=16)
         )
+
+    # test stac on year 2020
+    # handles resource description with and without ".tif" extension
+    def test_create_item_xml_ext(self):
```

Comment:
these two tests are doing the same operation on two different files and expecting the same results. They could be collapsed into one test, and looped like:

``
for filename in [
"data-files/m_3211605_ne_11_060_20200415_20200730.xml",
"data-files/m_3211605_ne_11_060_20200415_20200730_missing_resource_desc.xml"]:
  item = create_item(
            "ca",
            "2020",
            test_data.get_path("data-files/m_3211605_ne_11_060_20200415.tif"),
            test_data.get_path(filename),
        )
   ...
```